TFT/src/User/API/Touch_Screen.c

```diff
@@ -174,6 +175,7 @@ static inline uint8_t TS_CalibrationEnsure(uint16_t x, uint16_t y)
     GUI_DispStringCenter(LCD_WIDTH / 2, LCD_HEIGHT - 40, (int32_t)LABEL_ADJUST_FAILED);
     GUI_DispDec(0, 0, lcd_x, 3, 0);
     GUI_DispDec(0, 20, lcd_y, 3, 0);
+    Buzzer_AddSound(100, 200);
```

Comment:
move the buzzer call before the first GUI call and add an empty line before the first GUI call (separate GUI call from non GUI calls, when possible)